frontend/src/components/funding/FundingAgreementsTab.vue

```diff
@@ -51,7 +51,7 @@ export default {
         { title: 'Start Date', key: 'startDate' },
         { title: 'End Date', key: 'endDate' },
         { title: 'Status', key: 'status' },
```

Comment:
Not related to your change but it seems Status sort doesn't work.

Reply:
it is fixed